imports.pegjs

```diff
@@ -273,6 +277,53 @@ IsStatement
     }
   }
```

Comment:
I had to add the following statements to `imports.pegjs` in order to support the using statement.